geowebcache/diskquota/jdbc/src/main/resources/org/geowebcache/diskquota/jdbc/geowebcache-diskquota-jdbc.xsd

```diff
@@ -21,6 +21,12 @@
             <xs:documentation>JNDI resource used to locate the connection pool, optional, if this is missing then the connection pool element must be configured</xs:documentation>
           </xs:annotation>
         </xs:element>
+        <!-- Support for schema in configuration -->
+        <xs:element name="schema" type="xs:string" minOccurs="0">
+          <xs:annotation>
+            <xs:documentation>Database schema where the tables will be created. Defaults to public. Schema must exists.</xs:documentation>
```

Comment:
The value of the configuration parameter in GWC is not actually defaulting to `public` the DBMS is defaulting to whatever it defaults to, which in the case of Postgres is `public`.  Other DBMSes default other ways or lack this idea of a 'schema'.